spec/api-session-spec.ts

```diff
@@ -31,6 +31,16 @@ describe('session module', () => {
     });
   });
 
+  describe('session.fromPartition(partition)', () => {
+    const apppath = require('electron').app.getAppPath();
+    const loc = '/tmpstore';
+    const partionloc = apppath + loc;
+    const ses = session.fromPartition(partionloc);
```

Comment:
put this inside the `it` block, otherwise it is run at test runner startup time instead of at test time (so e.g. a failure here will crash at test boot time, regardless of whether this test is selected, instead of only crashing when the test is run.)

Reply:
Resolved in the new commit.